tasks/js-basics.md

```diff
@@ -60,6 +60,8 @@ When you finish this task you can proceed to the next one.
 - [Visualization of Common Algorithms and Data Structures](https://www.cs.usfca.edu/~galles/visualization/Algorithms.html)
 - [You Don't Know JS: Up & Going](https://github.com/getify/You-Dont-Know-JS/tree/master/up%20%26%20going) /
   [Вы не знаете JS: Начните и Совершенствуйтесь](https://github.com/azat-io/you-dont-know-js-ru/tree/master/up%20%26%20going)
+  [JavaScript Equality Table Game](https://eqeq.js.org/)
```

Comment:
Hi! Thank you very much for the contribution! Could you please fix missing newlines/list bullets? Here is what it looks like now:

![image](https://user-images.githubusercontent.com/17703790/64485867-505b4c80-d22e-11e9-9628-58c571f7b327.png)

Reply:
Hi!
THank you, I fixed it

Reply:
Thanks!